scipy/stats/stats.py

```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Tests for departure from
            normality", Biometrika, 60, 613-622
 
+    Examples
+    --------
+    >>> import scipy.stats as stats
+    >>> pts = 1000
+    >>> a = np.random.multivariate_normal([0], [[1]], size=pts)
```

Comment:
This uses `np.random.multivariate_normal()`, but it generates a one-dimensional sample, so you might as well use `np.random.normal(0, 1, size=pts)` instead.  (The same comment applies to the next line.)